### PR TITLE
(xpdf-utils) Update to 4.04

### LINKED ISSUE #2230: (xpdf-utils) Outdated
### Checklist

- [X] I have checked the moderation queue that no new version has been submitted (only visible when logged in), or the submitted package version has failed the automated checks.

Please navigate to the following [link](https://community.chocolatey.org/packages?q=&moderatorQueue=true&moderationStatus=all-statuses&prerelease=false&sortOrder=package-download-count) to view the moderation queue.

- [X] I have verified that the new version is a stable release.
- [X] I have looked to see if there are any Bug reports that prevents a new version being submitted and that no existing Outdated Reports have been created.
- [X] I have verified the new version has a Windows binary (EXE or MSI installer, Zip or other archive, etc).

### New Software Version

4.04

### Download location

https://www.xpdfreader.com/download.html

### Package Page

https://community.chocolatey.org/packages/xpdf-utils

### PR DESCRIPTION
**Pull request by brogers5**
## Description

This changeset should include the necessary changes to build an up-to-date version of the `xpdf-utils` package.

## Motivation and Context

Fixes #2230.

Unlike most packages maintained in this repository, this is a manually maintained package.

This includes some minor install script changes to reflect changes in the packaged archive's directory structure, as well as several metadata related changes, namely a new package icon and updates to project and package URLs.

#### Dev

* OS Build: Windows 11 Pro v10.0.22621.1702 (64-bit)
* PowerShell version: 7.3.4
* Chocolatey version: 1.4.0

#### Chocolatey Test Environment

* Vagrant Box Version: 3.0.0
* VM Provider: Virtualbox (although practically Hyper-V, since I manually converted the resulting virtual disk)
* OS Build: Windows Server 2019 Datacenter v10.0.17763.3887 (64-bit)
* PowerShell version: 5.1.17763.3770
* Chocolatey version: 1.4.0

### Steps

1. Executed `.\update.ps1` locally to pull in a copy of the packaged ZIP archive.
2. Executed `choco pack` to create a test package that consumes the proposed changes.
3. Within my local Chocolatey Testing Environment instance, installed the latest published `xpdf-utils` package version (3.4.0.20170430) from the Community Repository (i.e. `choco install xpdf-utils --source="'https://community.chocolatey.org/api/v2'"`).
4. Upgrade to the test package version (i.e. `choco upgrade xpdf-utils --source="'.'"`), and confirm the package upgrade completes without errors.
5. Browsed to `C:\ProgramData\chocolatey\lib\xpdf-utils\tools`, and confirmed the old package version's contents (i.e. the extracted `xpdfbin-win-3.04` directory) were removed.
6. Make a noticeable change to the extracted `xpdf-tools-win-4.04` directory to prepare a test of the modified cleanup filter pattern (e.g. rename the directory to `xpdf-tools-win-4.03`).
7. Force-reinstall the test package version (i.e. `choco install xpdf-utils --source="'.'" --force`), and confirm the package install completes without errors.
8. Browsed to `C:\ProgramData\chocolatey\lib\xpdf-utils\tools`, and confirmed the old package version's contents (i.e. the extracted and renamed `xpdf-tools-win-4.03` directory) were removed.
9. Uninstall the test package version (i.e. `choco uninstall xpdf-utils`), and confirm the package uninstall completes without errors.

## Screenshot (if appropriate, usually isn't needed):
N/A

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Migrated package (a package has been migrated from another repository)

## Checklist:
- [x] My code follows the code style of this repository.
- [ ] My change requires a change to documentation (this usually means the notes in the description of a package).
- [ ] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [ ] I have updated the package description and it is less than 4000 characters.
- [x] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey-community/chocolatey-packages/blob/master/CONTRIBUTING.md)
- [x] The added/modified package passed install/uninstall in the chocolatey test environment.
- [x] The changes only affect a single package (not including meta package).

### CODE REVIEW

**Comment by brogers5** [comment]
As an aside, is there any interest in converting this to an automatic package? There's been at least 6 software releases since the last packaged version (including this one) over the years. Seems to me like it should be one if the project is still being actively developed.

**Comment by pauby** [comment]
There may be reasons why this isn't an automatic package, but if nobody knows what those are, I don't see why it shouldn't be.

**Review by pauby — APPROVED** [review]

**Comment by pauby** [comment]
The changes look good to me. However, I don't know how we handle packages in this repository.

@AdmiringWorm how do we go about pushing manual packages?

File: manual/xpdf-utils/xpdf-utils.nuspec [hunk 1]
```diff
@@ -3,21 +3,22 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>xpdf-utils</id>
-    <version>3.4.0.20170430</version>
-    <packageSourceUrl>https://github.com/Chocolatey/chocolatey-coreteampackages/tree/master/automatic/xpdf-utils</packageSourceUrl>
+    <version>4.04</version>
```
Comment (AdmiringWorm) [inline manual/xpdf-utils/xpdf-utils.nuspec:6]: As we have started using Chocolatey CLI v2.0.0, please change this to be the normalized version number (the same as Chocolatey CLI will be using).

That means, no leading zeros, and a minimum of 3 part version numbers.
In this case that will be `4.4.0`.

**Comment by AdmiringWorm** [comment]
> The changes look good to me. However, I don't know how we handle packages in this repository.
> 
> @AdmiringWorm how do we go about pushing manual packages?

In general, the only think required is to create an empty commit with the message `[PUSH packageName]` to make it work. But it requires that everything is updated, and that the `update.ps1` script located next to the nuspec file sets up everything that isn't checked into the repository.

**Review by AdmiringWorm — CHANGES_REQUESTED** [review]
Just a small change, as this will be what Chocolatey CLI will be using.

**Comment by AdmiringWorm** [comment]
> There may be reasons why this isn't an automatic package, but if nobody knows what those are, I don't see why it shouldn't be.

There used to be a policy that any packages that hadn't received an update during 3 years would be moved to be a manual package (I am not sure if we still have that policy, as it was never documented). This could be one of the reasons, and another reason is that no one could be bothered to transform it to an automatic package when we migrated away from ketarin to use AU.

File: manual/xpdf-utils/xpdf-utils.nuspec (hunk 1 above)
Comment (brogers5) [reply manual/xpdf-utils/xpdf-utils.nuspec]: Addressed in 762fcd31673d90e0f42df9cf3f827c0e4ae5c4a3.

**Comment by pauby** [comment]
> In general, the only thing required is to create an empty commit with the message `[PUSH packageName]` to make it work. But it requires that everything is updated, and that the `update.ps1` script located next to the nuspec file sets up everything that isn't checked into the repository.

TIL. Manual packages have update scripts.

**Review by AdmiringWorm — APPROVED** [review]
LGTM

**Pull request merged**